tensorflow/lite/delegates/gpu/common/model_builder.cc

```diff
@@ -1392,6 +1393,8 @@ class MulOperationParser : public TFLiteOperationParser {
     const bool runtime_tensor0 = !constant_tensor0;
     const bool runtime_tensor1 = !constant_tensor1;
 
+    Node* node = graph->NewNode();
+
```

Comment:
Please, create a new node only when operation is created. If function returns with InternalError the graph will be left corrupted.

Reply:
I moved the params check a bit higher but it still seems that corruption is unavoidable.